app/workers/crawler/tabelog.rb

```diff
@@ -0,0 +1,44 @@
+class Crawler::Tabelog < Crawler::Base
+  include Crawler::Searchable
+
+  def initialize
+    super
+    @base_url = 'http://tabelog.com'
+    @site_name = 'tabelog'
+    @redis = Redis.new
+  end
+  def run
+    fetch_places
+    search_at_tabelog
+  end
+
+  def fetch_places
+    pages = []
+    tokyo_places = Place.where(prefecture_id: 13, place_type: 0)
+    tokyo_places.each do |place|
+      place_english_name = place.prefecture.name_e.downcase
+      place_japanese_name = URI.encode(place.address.match(/.*[区|市]/).to_s)
+      place_name = URI.encode(place.name)
+      tabelog_detail_page_url = "#{@base_url}/#{place_english_name}/rstLst/?vs=1&sa=#{place_japanese_name}&sk=#{place_name}&sw=#{place_name}"
```

Comment:
__[rubocop]__
* Line is too long. [141/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)